package.json

```diff
@@ -90,7 +91,9 @@
     "sinon-chai": "^2.8.0",
     "style-loader": "^0.13.2",
     "superagent": "^3.5.0",
-    "typescript": "4.0.2",
+    "ts-jest": "^26.4.1",
+    "ts-node": "^9.0.0",
+    "typescript": "^4.0.3",
```

Comment:
I usually avoid using `^` version specifier for TypeScript since they don't follow semantic versioning and might break things between minor versions. I prefer `4.0.x` etc to lock down minor versions. Not sure how important it is here though.